config/settings/base.py

```diff
@@ -282,6 +284,53 @@
 SOCIALACCOUNT_ALLOW_REGISTRATION = env.bool("DJANGO_SOCIALACCOUNT_ALLOW_REGISTRATION", True)
 ACCOUNT_ALLOWED_EMAIL_DOMAINS = env.list("DJANGO_ACCOUNT_ALLOWED_EMAIL_DOMAINS", default=["opencraft.com"])
 
+# Jira configs
+JIRA_SERVER = env.str("DJANGO_JIRA_SERVER")
```

Comment:
IMHO it's kind of confusing that the environment variable starts with `DJANGO_`. If it was all `DASHBOARD_` or just no prefix, I think it would be less confusing. `DJANGO_JIRA_SERVER` sounds like some kind of django-based proxy server that runs between this application and JIRA.